content/pages/team.html

```diff
@@ -115,6 +115,32 @@ <h1>Meet the Kiwi TCMS team</h1>
                 </a>
             </div> <!-- bio -->
 
+            <div class="member-bio">
+                <img class="rounded" src="/images/team/zahari.jpeg" height="150" width="150">
+
+                <strong>Zahari Zahariev</strong> -
+                <em>DevOps engineer</em>
```

Comment:
technical mentor - this is your position in the team. Daily job description can go below.